sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/PartitionReaderFactory.java

```diff
@@ -68,7 +68,7 @@ default boolean supportColumnarReads(InputPartition partition) {
   }
 
   /**
-   * Returns exact java types of the columns that are output in columnar processing mode.
+   * @return exact java types of the columns that are output in columnar processing mode.
```

Comment:
"Fully-qualified Java class names of"?

Reply:
> "Fully-qualified Java class names of"?

like this `org.apache.spark.sql.execution.vectorized.OnHeapColumnVector`

Reply:
`@return The fully-qualified Java class names of...`